admin-manual/installation/installation.rst

```diff
@@ -752,6 +752,12 @@ above, follow these steps to complete the configuration of your new server.
    *test* with the password *test*: http://localhost:8000 (or use the IP
    address of the machine you have been installing on).
 
+   If you are running the storage service and the dashboard on the same host you
+   should use 'localhost' or '127.0.0.1'
```

Comment:
The period is missing at the end of the sentence.
Nitpick: `s/storage service/Storage Service` and `s/dashboard/Dashboard`.

You may want to use [inline literals](http://docutils.sourceforge.net/0.6/docs/ref/rst/restructuredtext.html#inline-literals) for things like `localhost` or `127.0.0.1` or the ports below `80` and `8000`.